polkadot/cli/src/cli.rs

```diff
@@ -19,8 +19,15 @@
 use clap::Parser;
 use std::path::PathBuf;
 
-/// The version of the node. The passed-in version of the workers should match this.
-pub const NODE_VERSION: &'static str = env!("SUBSTRATE_CLI_IMPL_VERSION");
+/// The version of the node.
+///
+/// This is the version that is used for versioning this node binary.
+/// By default the `minor` version is bumped in every release. `Major` or `patch` releases are only
+/// expected in very rare cases.
+///
+/// The worker binaries associated to the node binary should ensure that they are using the same
+/// version as the main node that started them.
```

Comment:
hmm, how do they ensure that? wouldn't this fail due to https://github.com/paritytech/polkadot-sdk/blob/a414ea7515c9cdc81f1d12410e646afc148250e8/polkadot/src/bin/execute-worker.rs#L22

Reply:
:roll_eyes:

Reply:
Fixed